src/main/java/com/TableTalk/Enterprise/controllers/TableTalkController.java

```diff
@@ -1,14 +1,8 @@
 package com.TableTalk.Enterprise.controllers;
```

Comment:
I restructured this whole file by the mapping type, I think it makes more sense this way and increases readability.